ruma-events/src/room/message.rs

```diff
@@ -585,3 +590,15 @@ pub struct KeyVerificationRequestEventContent {
     /// events that have a m.reference relationship with this event.
     pub to: UserId,
 }
+
+/// The payload for a custom message event.
+#[doc(hidden)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct CustomEventContent {
+    /// A custom msgtype
+    pub msgtype: String,
+
+    /// Remaining event content
+    #[serde(flatten)]
+    pub data: serde_json::Map<String, JsonValue>,
```

Comment:
I guess I might have suggested this but after reviewing our other uses I think we can just use `BTreeMap` here too.